docs/conf.py

```diff
@@ -319,16 +319,17 @@
 linkcheck_workers = 32
 
 linkcheck_ignore = [
-    r"http://127.0.0.1(:\d+)?/?",
+    r"http://127\.0\.0\.1(:\d+)?/?",
```

Comment:
`.` is a meta character so we have to escape it, as also seen in the docs: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_allowed_redirects